source/comply/legal-hold.rst

```diff
@@ -43,86 +43,65 @@ Establish a policy for when to implement a legal hold. This should be developed
-2. Set either a **Global retention policy** that applies to all messages and files older in your system, or a **custom retention policy** for specific teams or channels.
-3. Learn more about data retention configuration options in our :doc:`product documentation </comply/data-retention-policy>`.
+You can optionally configure a :doc:`data retention policy </comply/data-retention-policy>` for the legal hold data you want to retain and delete to align with compliance requirements.
 
-Step 4: Identify custodians
+Step 4: Create a legal hold
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Select the individuals or user groups that you want to place on legal hold.
+In Mattermost, create a legal hold by completing the following steps:
 
-Step 5: Initiate legal hold
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Go to **System Console > Plugins > Legal Hold Plugin**, and select **Create new**.
+2. Enter a name for the legal hold.
+3. Specify the custodian user names of individuals or user groups that you want to place on legal hold.
+4. (Optional) You can choose to exclude public channels, if preferred.
+5. Specify the number of days custodians are placed in legal hold with a start and end date.
+6. Select **Create legal hold**.
 
-Use the built-in functionality in Smarsh to specify one or more custodians for legal hold, and the number of days the custodian(s) are placed on legal hold.
+While the legal hold is in place, you can edit it by adding or removing custodians, as well as download a copy of the preserved data to your local machine.
 
-While the legal hold is in place, you may edit it by adding or removing custodians, as well as access the preserved data.
+.. image:: ../images/legal-holds.png
+  :alt: An example of the Legal Hold management interface available to Mattermost system admins.
 
-Step 6: Release legal hold
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Step 5: Release a legal hold
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once the legal hold has completed, release it in Smarsh. This action will take custodians off of the legal hold.
+Once the legal hold has completed, you can release it to take custodians off of the legal hold by selecting the **Release** option to the right of the legal hold task.
 
 Frequently asked questions
 ---------------------------
 
 Who can implement legal hold?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Only Mattermost system administrators can implement a legal hold.
+Only Mattermost system admins can implement a legal hold.
 
 Does a user know if they're placed under a legal hold in Mattermost?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-No, users will not be notified if they're placed under a legal hold. This allows for investigations to be conducted without influencing user behavior and without conflicts of interest.
+No, users won't be notified if they're placed under a legal hold. This allows for investigations to be conducted without influencing user behavior and without conflicts of interest.
```

Comment:
I'm thinking we can be more definitive here, by saying users won't be notified, nor will any clue be present in their client application or via APIs.